Changed whitelist to allowlist and blacklist to denylist. (#21591)
kubernetes/website: content/en/docs/concepts/security/pod-security-standards.md

The Privileged policy is purposely-open, and entirely unrestricted. This type of policy is typically
aimed at system- and infrastructure-level workloads managed by privileged, trusted users.

> The privileged policy is defined by an absence of restrictions. For denylist-oriented enforcement

Reviewer: cc @tallclair - does this match your intent?

mechanisms (such as gatekeeper), the privileged profile may be an absence of applied constraints

> rather than an instantiated policy. In contrast, for a allowlist-oriented mechanism (such as Pod

Reviewer: cc @tallclair - does this match your intent?

				<br><b>Restricted Fields:</b><br>
				spec.containers[*].securityContext.capabilities.add<br>
				spec.initContainers[*].securityContext.capabilities.add<br>

> 				<br><b>Allowed Values:</b> empty (optionally allowlisted defaults)<br>

Reviewer: ```suggestion
				<br><b>Allowed Values:</b> empty (or restricted to a known list)<br>
```

		<tr>
			<td>Host Ports</td>
			<td>

> 				HostPorts should be disallowed, or at minimum restricted to a allowlist.<br>

Reviewer: ```suggestion
				HostPorts should be disallowed, or at minimum restricted to a known list.<br>
```

				<br><b>Restricted Fields:</b><br>
				spec.containers[*].ports[*].hostPort<br>
				spec.initContainers[*].ports[*].hostPort<br>

> 				<br><b>Allowed Values:</b> 0, undefined, (allowlisted)<br>

Reviewer: ```suggestion
				<br><b>Allowed Values:</b> 0, undefined (or restricted to a known list)<br>
```

			</td>
		</tr>
		<tr>
			<td>AppArmor <em>(optional)</em></td>
			<td>

> 				On supported hosts, the 'runtime/default' AppArmor profile is applied by default. The default policy should prevent overriding or disabling the policy, or restrict overrides to a allowlisted set of profiles.<br>

Reviewer: ```suggestion
				On supported hosts, the 'runtime/default' AppArmor profile is applied by default. The default policy should prevent overriding or disabling the policy, or restrict overrides to an allowed set of profiles.<br>
```

		<tr>
			<td>Sysctls</td>
			<td>

> 				Sysctls can disable security mechanisms or affect all containers on a host, and should be disallowed except for a allowlisted "safe" subset.

Reviewer: ```suggestion
				Sysctls can disable security mechanisms or affect all containers on a host, and should be disallowed except for an allowed "safe" subset.
```

		<tr>
			<td>Seccomp</td>
			<td>

> 				The 'runtime/default' seccomp profile must be required, or allow additional allowlisted values.<br>

Reviewer: ```suggestion
				The 'runtime/default' seccomp profile must be required, or allow specific additional profiles.<br>
```
